### ISSUE TITLE
Bug: helm history causes SIGSEGV

### ISSUE THREAD

**Issue by thomasbrezinski** [kubernetes/helm#1347]
Trying to fetch release history for a chart with helm v2.0.0-alpha.5

```
helm version
Client: &version.Version{SemVer:"v2.0.0-alpha.5", GitCommit:"36ec1901ebd68ae976fa8ef332ca5ed44598fc24", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.0.0-alpha.5", GitCommit:"4de5c4aa578ed9ca173aff472f00d2167ec59a88", GitTreeState:"dirty"}
```

```
helm history stable/mysql
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8b76]

goroutine 1 [running]:
panic(0x176fae0, 0xc420014080)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
main.formatHistory(0xc4205a0680, 0x3, 0x4, 0xc42002c5a8, 0x1)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/cmd/helm/history.go:101 +0x326
main.(*historyCmd).run(0xc4200781c0, 0x1, 0x1)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/cmd/helm/history.go:91 +0xf4
main.newHistoryCmd.func1(0xc42048f8c0, 0xc4203b6a50, 0x1, 0x1, 0x0, 0x0)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/cmd/helm/history.go:70 +0x6c
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).execute(0xc42048f8c0, 0xc4203b6a00, 0x1, 0x1, 0xc42048f8c0, 0xc4203b6a00)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:571 +0x234
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc42009cd80, 0xc4203c26c0, 0xc4203c2900, 0xc42009cfc0)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:661 +0x367
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).Execute(0xc42009cd80, 0xc420080008, 0xc42009cd80)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:620 +0x2b
main.main()
	/home/ubuntu/.go_workspace/src/k8s.io/helm/cmd/helm/helm.go:120 +0x47
```

**Comment by technosophos** [comment]
Thanks. I see what's going on there. It looks like one of the releases is missing the data for its Chart.yaml file. I'll fix this issue. After that, I may ask for some help tracking down the release that is missing the data.

**Comment by technosophos** [comment]
Closed by #1349
